beater/config/config.go

```diff
@@ -114,7 +114,8 @@ func NewConfig(ucfg *common.Config, outputESCfg *common.Config) (*Config, error)
 	}
 
 	if err := c.JavaAttacherConfig.setup(); err != nil {
-		return nil, err
+		logger.Errorf("failed to setup java-attacher: %v", err)
```

Comment:
I'd probably make it a warning, which (to me at least) implies that it's not going to stop the server from functioning generally.

Reply:
Makes sense, thanks!